src/transformers/models/audio_spectrogram_transformer/modeling_audio_spectrogram_transformer.py

```diff
@@ -481,7 +481,7 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
-    ):
+    ) -> BaseModelOutputWithPooling:
```

Comment:
I think if `return_dict` is False you might get a tuple instead, but other than that this looks great!
```suggestion
    ) -> Union[Tuple, BaseModelOutputWithPooling]:
```

Reply:
Ahh, right, so silly mistake :facepalm: